ts_src/psetv2/updater.ts

```diff
@@ -306,6 +319,7 @@ export class Updater {
     pset.inputs[inIndex].issuanceInflationKeys = tokenAmount;
     pset.inputs[inIndex].issuanceAssetEntropy = issuance.assetEntropy;
     pset.inputs[inIndex].issuanceBlindingNonce = issuance.assetBlindingNonce;
+    pset.inputs[inIndex].blindedIssuance = args.blindedIssuance ? true : false;
```

Comment:
the spec says:
> A boolean flag. 0x00 indicates the issuance should not be blinded, 0x01 indicates it should be. If not specified, assumed to be 0x01. Note that this does not indicate actual blinding status, but rather the expected blinding status prior to signing. 

This therefore means that the default value for this flag is TRUE even if not defined, and false only if expressively defined.

Also, `IssuanceOpts.blindedIssuance` must be an optional field